proto/viam/component/movementsensor/v1/movementsensor.proto

```diff
@@ -9,6 +9,20 @@ import "google/protobuf/struct.proto";
 option go_package = "go.viam.com/api/component/movementsensor/v1";
 option java_package = "com.viam.component.movementsensor.v1";
 
+// The meaning of each fix value is documented here
+// https://docs.novatel.com/OEM7/Content/Logs/GPGGA.htm#GPSQualityIndicators
```

Comment:
Add a bit more information in the comment about what GGA is and what. fix is.

Reply:
Also maybe that the most common are 1 and 2 and are bad fixes, and 4 and 5 are what we actually want to have